kong/plugins/log-serializers/basic.lua

```diff
@@ -16,6 +16,7 @@ function _M.serialize(ngx)
   return {
     request = {
       uri = ngx.var.request_uri,
+      new_uri = ngx.var.uri,
```

Comment:
@pablofmorales , correct. But the value of `$uri` can change - for example during redirects, which is not captured by `$request_uri`
@thibaultcha I agree that for most cases, both `ngx.var.request_uri` and `ngx.var.uri` would be same.  Hence, renaming them could lead to confusion. In fact, this would only be useful if there is some redirect or uri modification happening and I want to log those as well. Let's use the same name as option 1 suggests. However, then we would have to modify the **name** of the field `request_uri` too. Maybe `request_url` ? :)
```
request_uri = ngx.var.request_uri,
uri = ngx.var.uri,
request_url = ngx.var.scheme.."://"..ngx.var.host..":"..ngx.var.server_port..ngx.var.request_uri 
```
I will add test cases, thanks.